examples/README.adoc

```diff
@@ -63,7 +63,8 @@ The following examples are provided:
 	is generated using U2F (CTAP1) instead of FIDO2 (CTAP2) commands.
 	The -T option may be used to enforce a timeout of <seconds>. If the
 	option -b is specified, the credential's "largeBlob" key is stored in
-	<blobkey>.
+	<blobkey>. If the option -c is specified the credProtect extension
+	will be added to the credential with the choosen policy.
```

Comment:
s/choosen/chosen/? It also seems easy to misinterpret `added to the credential` as something that can happen after the credential is created. Something along the lines of `the generated credential will be bound by the specified protection policy` would arguably be more accurate.